sqlalchemy_jsonapi/unittests/main.py

```diff
@@ -0,0 +1,16 @@
+"""Must set up flask app for sqlalchmey_jsonapi unittesting."""
+
+from flask import Flask
```

Comment:
Is `sqlalchemy_jsonapi` actually using flask? Is this even necessary?

(If yes, I guess our first goal is to make it...not?)